static/src/stylesheets/module/content/_live-blog.scss

```diff
@@ -478,12 +478,11 @@ $block-padding-right: $gs-gutter;
 .dropdown--live-feed {
     border-top: 0;
 
+    $content-gutter: $gs-gutter / 2;
```

Comment:
Strange too as it's not a complete abstraction i.e. `($gs-gutter / 2)*-1` which makes the abstraction confusing in itself?